.github/workflows/php-test.yml

```diff
@@ -72,7 +80,7 @@ jobs:
       - name: Setup PHPStan
         uses: shivammathur/setup-php@v2
         with:
-          php-version: '7.4'
+          php-version: '8.1'
```

Comment:
I'm afraid I have to ask to keep this at 7.4 after all. With this neither the phpcs nor the phpstan checks execute anymore, while they do just fine in `develop`.

Reply:
Alternatively, we can bump the versions in `composer.json`. That's likely more work though.

Reply:
#88 :slightly_smiling_face: